build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ls -op packer | grep -v / | awk '{print $8}' | tail -n +2 | while read image
+
+do
+    packer build packer/$image
+done
```

Comment:
I think we should sit down and discuss this more. I'm not sure this is the right approach.